yoyodyne/train.py

```diff
@@ -224,52 +227,60 @@ def get_model(
     decoder_layers: int = defaults.DECODER_LAYERS,
     embedding_size: int = defaults.EMBEDDING_SIZE,
     encoder_layers: int = defaults.ENCODER_LAYERS,
+    feature_encoder_arch: Optional[str] = None,
     hidden_size: int = defaults.HIDDEN_SIZE,
     max_source_length: int = defaults.MAX_SOURCE_LENGTH,
     max_target_length: int = defaults.MAX_TARGET_LENGTH,
+    source_encoder_arch: str = None,
     # Training arguments.
     batch_size: int = defaults.BATCH_SIZE,
     beta1: float = defaults.BETA1,
     beta2: float = defaults.BETA2,
     dropout: float = defaults.DROPOUT,
+    label_smoothing: Optional[bool] = None,
```

Comment:
this is wrong, `label_smoothing` is not applied if null and otherwise it's a float in (0, 1)...

Reply:
So should default be 0 for null or `None`?